docs/writing-docs/docs-page.md

```diff
@@ -1,205 +1,189 @@
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-![Subcomponents in Docs Page](./docspage-subcomponents.png)
-
-Subcomponent `ArgsTables` will show up in a tabbed interface along with the primary component. The tab titles will correspond to the keys of the subcomponents object.
-
-If you want to organize your documentation differently for component groups, we recommend using MDX. It gives you complete control over how it's displayed and supports any configuration.
-
-## Replacing DocsPage
-
-Replace the DocsPage template with your own to customize its contents.
+| Option        | Description                                                                                            |
+| ------------- | ------------------------------------------------------------------------------------------------------ |
+| `enabled`     | Toggles support for auto-generated documentation pages <br/> `docs: { enabled:false }`                 |
+| `docsPage`    | Disables auto-generated documentation pages created via `tags` <br/> `docs: { docsPage: false }`       |
+| `automatic`   | Enables auto-generated documentation pages for every component <br/> `docs: { docsPage: 'automatic' }` |
+| `defaultName` | Renames the auto-generated documentation page<br/> `docs: { defaultName: 'Documentation' }`            |
 
-### With null to remove docs
+### Write a custom template
 
-Override the `docs.page` [parameter](../writing-stories/parameters.md) with `null` to remove its contents.
+To replace the default documentation templated used by Storybook, you can create your own template written in MDX and update your UI configuration file (i.e., `.storybook/preview.js`) to point to it.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/button-story-disable-docspage-component.js.mdx',
+    'common/storybook-preview-auto-docs-custom-template.js.mdx',
+    'common/storybook-preview-auto-docs-template.mdx.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-### With MDX documentation
+<div class="aside">
 
-Write your documentation in MDX and update the `docs.page` [parameter](../writing-stories/parameters.md) to display it. The `id` of reference follows the pattern: `group-subgroup-...--name`, where the `groups` and `subgroups` are defined as according to the [Grouping Documentation](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#grouping).
+💡 If you only need to override the documentation page for a single component, we recommend creating an MDX file and referencing it directly via the `<Meta of={} />` Doc Block.
+
+</div>
+
+## Setup custom documentation
+
+Storybook provides support for MDX 2, an open standard markup language, combining two other ones: Markdown, which is used for formatting text, and JSX, which is used for rendering dynamic elements on a page. To enable custom documentation for your stories with this format, start by updating your Storybook configuration file (i.e., `.storybook/main.js|ts|cjs`).
```

Comment:
> To enable custom documentation for your stories with this format, start by updating your Storybook configuration file

Doesn't _any_ use of MDX in SB use MDX 2? This seems a little misleading, and perhaps this intro to MDX should be higher up this document?